matchms/filtering/remove_spectra_below.py

```diff
@@ -0,0 +1,24 @@
+from ..typing import SpectrumType
+
+
+def remove_spectra_below(spectrum: SpectrumType, max_mz: float = 1000) -> SpectrumType:
```

Comment:
```suggestion
def remove_spectra_above_mz(spectrum: SpectrumType, max_mz: float = 1000) -> SpectrumType:
```

Reply:
``remove_spectra_below`` sounded misleading to me. Maybe we could also use ``remove_spectra_above_precursor_mz`` but that is getting a bit long...

Reply:
Yes I agree, the previous name was quite misleading. `remove_spectra_below ` sounds fine to me.